frame/bounties/src/benchmarking.rs

```diff
@@ -33,7 +33,8 @@ const SEED: u32 = 0;
 // Create bounties that are approved for use in `on_initialize`.
 fn create_approved_bounties<T: Config>(n: u32) -> Result<(), &'static str> {
 	for i in 0..n {
-		let (caller, _curator, _fee, value, reason) = setup_bounty::<T>(i, MAX_BYTES);
+		let (caller, _curator, _fee, value, reason) =
+			setup_bounty::<T>(i, T::MaximumReasonLength::get());
```

Comment:
maybe we should add in the doc of MaximumReasonLength that this value is used in benchmark, so changing it requires running the benchmark again.
(so it can't be changed without runtime upgrade).